services/npm/npm-rating.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const Joi = require('joi')
+const { starRating } = require('../text-formatters')
+const { BaseJsonService } = require('..')
+
+// https://api-docs.npms.io/#api-Package-GetPackageInfo
+const responseSchema = Joi.object({
+  score: Joi.object({
+    final: Joi.number().required(),
+    detail: Joi.object({
+      maintenance: Joi.number().required(),
+      popularity: Joi.number().required(),
+      quality: Joi.number().required(),
+    }),
+  }),
+}).required()
+
+module.exports = class NpmRating extends BaseJsonService {
+  static category = 'rating'
+
+  static route = {
+    base: 'npm',
+    pattern:
+      'rating/:type(maintenance|popularity|quality)?/:scope(@.+)?/:packageName',
+  }
+
+  static examples = [
+    {
+      title: 'NPM Rating',
```

Comment:
The title should reflect the same name changes discussed above, and be `npms` or `npms.io` for a bigger visual distinction from the `npm` badges

Reply:
change to `npms.io`